include/sound/sof/header.h

```diff
@@ -74,6 +74,7 @@
 #define SOF_IPC_PM_CLK_GET			SOF_CMD_TYPE(0x005)
 #define SOF_IPC_PM_CLK_REQ			SOF_CMD_TYPE(0x006)
 #define SOF_IPC_PM_CORE_ENABLE			SOF_CMD_TYPE(0x007)
+#define SOF_IPC_PM_GATE				SOF_CMD_TYPE(0x008)
```

Comment:
'GATE' can mean CLK_GATE or POWER_GATE, so this definition is very very ambiguous.
And it's also unclear if this is for a core or for the entire IP.

Reply:
I think it is for the entire IP, we have different flag bits for clk_gate and power_gate, as added in pm.h